ReadMeDay2.md

```diff
@@ -0,0 +1,88 @@
+- implement media queries in a project.
+- describe how using scalable units for font sizes impacts accessibility and why designing accessible web pages is important
+
+## Introduction
+
+It goes without saying that this skill is essential for a professional Web Developer. Imagine trying to visit a store's webpage on your phone and not being able to even read the names of items? That would be totally unacceptable in this day and age! Same goes for a web developer's portfolio site.
+
+In this challenge you will refactor your personal portfolio code to make it responsive. You have the ability to write HTML, CSS, and responsive media queries. You also know how to identify and write responsive units. It's time to put that knowledge into action!
+
+## Instructions
+
+### Task 1: Set up Project
+
+New features of a project should occur in a new branch in the same repository. Follow these steps to set up and work on your project from yesterday:
+
+- [x] `cd` into your personal portfolio folder
+- [x] Create a new branch from `<firstName-lastName>`:
+
+```bash
+git checkout -b <firstName-lastName-day2> <firstName-lastName>.
+```
+
+- [x] Implement the project on your newly created `<firstName-lastName-day2>` branch, committing changes regularly.
+- [x] Push commits: git push origin `<firstName-lastName-day2>`.
+
+### Task 2: Minimum Viable Product
+
+Use the checklist below to guide your work today. Your final product should be presentable at mobile, tablet and a desktop-width.
+
+- [x] Insert a viewport meta tag into the head of the project with these html attributes: content="width=device-width, initial-scale=1"
+- [x] Introduce max-width media queries into your project at 800px and 500px
+- [x] Add accessability features to your webpage
+- [x] Design should closely follow the [mobile wireframe](Wireframes/) given for your chosen layout
+
+### Task 2b: Exit Ticket
+
+Once you begin, you will have 15 minutes to answer the questions [here](https://app.codesignal.com/public-test/hALhPTQrbZCJtPYhn/F78eZwAGpSS3cR)
+
+The completion of these questions is mandatory for MVP. However, passing the quiz doesn't affect your standing as a Lambda School student whatsoever. This is Lambda School testing itself! Please answer honestly and to the best of your ability without using external references.
+
+### Task 3: Stretch Goals
+
+Once you finish the minimum viable project, work on any of the following stretch goals:
+
+- [x]  Test your website at several breakpoints and refactor code as needed. A few common breakpoints are below:
+  - [x]  iPhone: 360×640
+  - [x]  Laptop: 1366×768
+  - [x]  Widescreen: 1920×1080
+- [x] Test your webpage's accessibility with a screen reader like [this](https://support.google.com/accessibility/answer/7031755?hl=en)
+- [ ] Start over with min-width media queries to get a feel for how a mobile first approach would be like. I recommend making a branch of all your content in a new folder named "mobile-first" to keep it separate
```

Comment:
If you get time, I would encourage you to do this. Mobile-first is usually easier to picture in your head and usually needs less CSS to accomplish it. Given a fresh start, I always work this way.